src/components/Thanks/SingleVersion/ControlModule.vue

```diff
@@ -0,0 +1,257 @@
+<template>
+	<div class="module-container">
+		<div
+			style="min-height: 481px;"
+			class="
+				tw-flex
+				tw-flex-col
+				tw-items-center
+				tw-gap-1.5
+				tw-text-center
+				tw-overflow-hidden
+				print:tw-hidden
+				tw-relative
+				tw-justify-between
+			"
+		>
+			<h2 v-html="title" class="tw-text-center"></h2>
+			<!-- BG Rays -->
```

Comment:
If the rays are identical to the badge component, can we extract this our and share between the two?